components/dataViews/ThresholdInfo.tsx

```diff
@@ -17,7 +17,8 @@ const ThresholdInfo = ({ signatures, pubkey }: Props) => {
     setUrlToCopy(urlWithoutQuery);
   }, []);
 
-  const remainingSignatures = Number(pubkey.value.threshold) - signatures.length;
+  const signaturesDiff = Number(pubkey.value.threshold) - signatures.length;
+  const remainingSignatures = signaturesDiff > 0 ? signaturesDiff : 0;
   if (!remainingSignatures) return null;
```

Comment:
Two suggestions here that might help improve the code:

```suggestion
  const missingSignatures = Math.max(Number(pubkey.value.threshold) - signatures.length, 0);
  if (!missingSignatures) return null;
```